moped-editor/src/views/projects/projectView/ProjectView.js

```diff
@@ -455,20 +454,16 @@ const ProjectView = () => {
                             display="flex"
                             flexDirection="row"
                           >
-                            <ProjectNameEditable
-                              projectName={data.moped_project[0].project_name}
+                            <ProjectNameComplex
+                              projectData={data.moped_project[0]}
                               projectId={projectId}
                               editable={true}
                               isEditing={isEditing}
                               setIsEditing={setIsEditing}
-                              updatedCallback={handleNameUpdate} // FLH not sure if this is needed
+                              updatedCallback={handleNameUpdate}
+                              currentPhase={currentPhase}
                             />
-                            <Box>
-                              <ProjectStatusBadge
-                                phaseKey={currentPhase?.phase_key}
-                                phaseName={currentPhase?.phase_name}
-                              />
-                            </Box>
```

Comment:
This is pulled into the `ProjectNameComplex` component so that I can continue to have it float to the left up against the project name and number.

Reply:
I agree with Mike, on both counts. It would be nice to refactor it so that we can keep the badge separate from the name. But could be a spin off issue.

Reply:
https://github.com/cityofaustin/atd-data-tech/issues/16776